src/requestHandler.ts

```diff
@@ -267,7 +268,9 @@ export default class RequestHandler {
       return;
     }
 
-    await this.app.vault.adapter.write(path, req.body);
+    const directory = path.relative(process.cwd(), path.dirname(filepath))
```

Comment:
I'm not totally sure I understand what you're trying to accomplish by calculating the relative path between the process's current working directory (which could be anywhere on the filesystem -- even somewhere unrelated to the path to your vault) and the directory name of the file you're sending in.  Care to elaborate?

The reason I ask is that I believe `this.app.vault.adapter.exists(path.dirname(filepath))` would be sufficient, no?  I'm under the impression that the vault adapter already assumes that file paths passed-in are relative to your vault root.

Reply:
Seems that I facing a problem is the dirname will return a full path. But now it's not. I have no idea what is happened. But I removed that now